frontend/src/component/application/ApplicationIssues/ApplicationIssues.tsx

```diff
@@ -230,7 +230,10 @@ export const ApplicationIssues = ({ application }: IApplicationIssuesProps) => {
             ),
         ),
     ];
-    const issueCount = mode.issueCount;
```

Comment:
what about the code in the getApplicationIssueMode that was giving incorrect results? Should we delete it or move the new fixed code there?